operator/charts/kit-operator/templates/data-plane-crd.yaml

```diff
@@ -35,10 +35,35 @@ spec:
             type: object
           spec:
             properties:
+              allocationStrategy:
```

Comment:
These additions to CRD forces us to change in KIT tekton specs here - https://github.com/awslabs/kubernetes-iteration-toolkit/blob/main/tests/tasks/setup/kit-cluster/default.yaml#L38-L95

Reply:
We will have to update the spec anyways to include the subnet selector and most of this changes are adding description except the subnetSelector being a new field.

Reply:
do you have an example spec added in example folder leveraging all these new fields to refer for Tekton tasks  and also will help other customers ?

Reply:
Yesh its there - https://github.com/awslabs/kubernetes-iteration-toolkit/blob/main/operator/docs/examples/dataplane.yaml#L8